### PR TITLE
Login navbar

### PR DESCRIPTION
**Pull request by syboulette**

### CODE REVIEW

File: app/controllers/users/registrations_controller.rb [hunk 1]
```diff
@@ -10,6 +10,9 @@ def create
         fridge = Fridge.new(name: 'My Fridgy')
         fridge.user = resource
         fridge.save!
+        list = List.new(name: 'My Fridgy')
```
Comment (Za-hell) [inline app/controllers/users/registrations_controller.rb:13]: My last branch i've updated the name to My Fridgi

**Review by Za-hell — COMMENTED** [review]
Would we like to use 'My Fridgi" name at registrations_controller.rb ?

File: app/controllers/users/registrations_controller.rb (hunk 1 above)
Comment (syboulette) [reply app/controllers/users/registrations_controller.rb:13]: Sorry, forgot to split my branch did not finish the list part :)

**Review by Za-hell — APPROVED** [review]
:)

**Review by D-Zaleski — APPROVED** [review]

**Pull request merged**